### PR TITLE
Rewrite tests with Jest

### PR DESCRIPTION
**Pull request by hulkoba**
Say hello to Jest :wave:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -66,9 +69,13 @@
...
     "pretest": "standard && npm run db:start",
     "start": "node index.js",
-    "test": "NODE_ENV=testing nyc tap test"
+    "test": "NODE_ENV=testing jest -i"
   },
   "standard": {
+    "env": {
+      "jest": true
+    },
+    "globals": [ "jest", "expect", "describe", "test", "beforeAll", "beforeEach", "afterAll", "aftereach"],
```
Comment (Realtin) [inline package.json]: "aftereach" => "afterEach" ?

File: test/jobs/cancel-stripe-subscription.js [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
   const { payments } = await dbs()
-  t.plan(3)
+  expect.assertions(3)
 
   nock('https://api.stripe.com/v1')
     .delete('/subscriptions/345')
     .reply(200, () => {
-      t.pass('Stripe called')
+      // Stripe called
+      expect(true).toBeTruthy()
```
Comment (Realtin) [inline test/jobs/cancel-stripe-subscription.js]: maybe we should replace those  ` expect(true).toBeTruthy()` with `.pass`  from https://github.com/jest-community/jest-extended

File: test/jobs/create-initial-pr-comment.js [hunk 3]
```diff
@@ -85,98 +58,204 @@ test('create-initial-pr-comment', async t => {
...
+      .reply(200, {
+        name: 'test',
+        html_url: 'https://github.com/finnp/test',
+        newBranch: 'greenkeeper/initial'
+      })
       .get('/repos/finnp/test/issues/1234')
       .reply(200, () => {
-        t.fail('Should not query issue status')
+        // Should not query issue status
+        expect(false).toBeFalsy()
```
Comment (Realtin) [inline test/jobs/create-initial-pr-comment.js]: same here, could be replaced with https://github.com/jest-community/jest-extended `.fail`

File: test/jobs/create-initial-branch.js [hunk 4]
```diff
@@ -1,47 +1,48 @@
...
 const removeIfExists = require('../helpers/remove-if-exists')
 const { cleanCache } = require('../helpers/module-cache-helpers')
 
-const dbs = require('../../lib/dbs')
+nock.disableNetConnect()
+nock.enableNetConnect('localhost')
 
-test('create-initial-branch', async t => {
-  t.beforeEach(() => {
+describe('create initial brach', () => {
```
Comment (espy) [inline test/jobs/create-initial-branch.js]: Typo: brach -> branch

File: test/jobs/create-initial-pr-comment.js [hunk 5]
```diff
@@ -1,53 +1,28 @@
...
 const nock = require('nock')
-const worker = require('../../jobs/create-initial-pr-comment')
-const upsert = require('../../lib/upsert')
-
 const dbs = require('../../lib/dbs')
-const timeToWaitAfterTests = 500
+const upsert = require('../../lib/upsert')
+const removeIfExists = require('../helpers/remove-if-exists')
 
+const timeToWaitAfterTests = 50
```
Comment (espy) [inline test/jobs/create-initial-pr-comment.js]: Why 50 instead of 500?

**Pull request merged**